### ISSUE TITLE
[improvement] Add an Archive button in detailed record view

### ISSUE THREAD

**Issue by asitemade4u** [sissbruecker/linkding#193]
Sometimes, when coming back to an old bookmark, one finds it irrelevant.
Adding an Archive button to the detailed view of a bookmark (the one you get when clicking on the **Edit** link) would help.

**Comment by sissbruecker** [comment]
@asitemade4u Can you elaborate on the use-case? Do you actually edit bookmarks to review them? Why, is there any additional information that you don't get in the overview (I can't think of any)?

Otherwise this is a duplicate of https://github.com/sissbruecker/linkding/issues/82 which was closed by the author, so I might want to reopen that one instead.

**Comment by asitemade4u** [comment]
Yes it is indeed a duplicate of #82 . The idea is to be able to archive a document from the detail view instead of having to:
1. go back to the list
2. click on the **Archive** link

**Comment by sissbruecker** [comment]
OK then, closing this in favor of https://github.com/sissbruecker/linkding/issues/82
